src/edu/csus/ecs/pc2/core/StringUtilities.java

```diff
@@ -505,4 +506,86 @@ else if (lastNumericCompare == 0)
         else
             return aLength - bLength;
     }
+
```

Comment:
Again, I'm not 100% certain this is where these routines belong, since we really don't have a "ClicsUtils", and this is very specific string checking, so I think it's OK for now.  This will eventually be deprecated when we read the configuration from JSON files anyway.  No reply necessay.

Reply:
Perhaps we SHOULD have a 'ClicsUtils' class? Or a 'Clics' package?